### ISSUE TITLE
Missing mathmap files in 2.7.8

### ISSUE THREAD

**Issue by garagebrewer** [mathjax/MathJax#2436]
I am trying to test some accessibility features in 2.7.8 but there are several errors during loading.  It requests several mathmaps that are missing/don't exist.  I wonder if this is just a version control thing.

Steps to recreate:
1. Clone the project
2. git checkout 2.7.8
3. Use a configuration that enables the explorer
4. Look in the console.  Several missing files coming from mathjax-sre

Example:
MathJax.Hub.Config({
	menuSettings: {
		collapsible: false,
		autocollapse: false,
		explorer: true
	}
});

Explorer "on" reveals the problem: (https://content.nroc.org/MathJax.2.7.8/mytest.html)
Explorer "off" has no problem: (https://content.nroc.org/MathJax.2.7.8/mytest2.html)

**Comment by dpvc** [comment]
Thanks for the report.  It appears that I did not handle the update to SRE 2.4.0 properly in the 2.7.8 release (I guess I've gotten too used to the v3 toolchain that handles these things automatically).  I will need to make a 2.7.9 release to fix that.
